.travis.yml

```diff
@@ -1,7 +1,9 @@
 ---
 language: python
 cache: pip
-python: "2.7"
+python:
+  - "2.7"
+  - "3.6"
```

Comment:
We should use 3.7 here

Reply:
Alright, we should still test with 3.6 though because this is the default version on CentOS 8.